### PR TITLE
add Flotiq benchmark code

### PR DESCRIPTION
**Pull request by andrzejwp**
## Description

Introduce Flotiq support in the willit.build benchmark.

### Documentation
2 standard modes of operation are supported, as per Gatsby dosc:

- `gatsby build`
- `npm run data-update`

and the following dataset sizes are supporter:
- 512
- 4096
- 8192
- 32768

access keys have been provided to Gatsby via e-mail.

### CODE REVIEW

File: benchmarks/source-flotiq/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "gatsby-transformer-sharp": "^2.3.14",
+    "lodash.kebabcase": "^4.1.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
+  },
+  "devDependencies": {
+    "cross-env": "^7.0.0",
+    "prettier": "^1.19.1"
+  },
+  "repository": {
```
Comment (moonmeister) [inline benchmarks/source-flotiq/package.json:34]: This can be updated or removed

File: benchmarks/source-flotiq/gatsby-browser.js [hunk 2]
```diff
@@ -0,0 +1 @@
+import "./styles.css"
```
Comment (moonmeister) [inline benchmarks/source-flotiq/gatsby-browser.js:1]: Nit: I would move this to me imported into your layout. It's possible this works fine, I've just never seen this done. I am also wondering if there maybe be some strange side effects happen, if it works at all. 🤷‍♂️

File: benchmarks/source-flotiq/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "type": "git",
+    "url": "https://github.com/gatsbyjs/gatsby-starter-hello-world"
+  },
+  "bugs": {
```
Comment (moonmeister) [inline benchmarks/source-flotiq/package.json:38]: This can be updated or removed

File: benchmarks/source-flotiq/package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "gatsby-starter-hello-world",
```
Comment (moonmeister) [inline benchmarks/source-flotiq/package.json:2]: Probably worth updating this name and the description

File: benchmarks/source-flotiq/gatsby-config.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      },
+    },
+    `gatsby-transformer-sharp`,
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `pages`,
+        path: `${__dirname}/src/pages/`,
+      },
+    },
```
Comment (moonmeister) [inline benchmarks/source-flotiq/gatsby-config.js:25]:
```suggestion
```

I'm not sure why this is here, gatsby handles pages in `src/pages` automatically. What was the reason this was put here?

File: benchmarks/source-flotiq/package.json (hunk 1 above)
Comment (moonmeister) [reply benchmarks/source-flotiq/package.json:34]: "This" being the repository reference

File: benchmarks/source-flotiq/package.json (hunk 3 above)
Comment (moonmeister) [reply benchmarks/source-flotiq/package.json:38]: "This" being the bugs url

File: benchmarks/source-flotiq/gatsby-config.js (hunk 5 above)
Comment (andrzejwp) [reply benchmarks/source-flotiq/gatsby-config.js]: Copied it from one of the examples. Removed along with other updates in the next commit.

File: benchmarks/source-flotiq/package.json (hunk 4 above)
Comment (andrzejwp) [reply benchmarks/source-flotiq/package.json]: Done, thanks for pointing out.

File: benchmarks/source-flotiq/package.json (hunk 3 above)
Comment (andrzejwp) [reply benchmarks/source-flotiq/package.json]: Done, thanks for pointing out.

File: benchmarks/source-flotiq/gatsby-browser.js (hunk 2 above)
Comment (andrzejwp) [reply benchmarks/source-flotiq/gatsby-browser.js]: Done, thanks for pointing out.

File: benchmarks/source-flotiq/package.json (hunk 1 above)
Comment (andrzejwp) [reply benchmarks/source-flotiq/package.json]: Done, thanks for pointing out.

**Comment by andrzejwp** [comment]
Hi @moonmeister, have you had a chance to review the amendments? Also - if you could let me know when your team will be able to test.

**Comment by duffn** [comment]
@andrzejwp We have not yet had the opportunity to test, but will certainly keep you updated as to when we do.

**Comment by andrzejwp** [comment]
Hi @duffn, any news on this? Can I help to get this merged?

File: benchmarks/source-flotiq/gatsby-config.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
```
Comment (duffn) [inline benchmarks/source-flotiq/gatsby-config.js:1]: Can you please use `NODE_ENV` in the `dotenv` config as done here? https://github.com/gatsbyjs/gatsby/blob/cf3430462748cecdfb4b9f6b61285534fb4a2ea9/benchmarks/source-contentful/gatsby-config.js#L1

File: benchmarks/source-flotiq/scripts/data-update.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+require(`dotenv`).config();
```
Comment (duffn) [inline benchmarks/source-flotiq/scripts/data-update.js:3]: Can you please use `NODE_ENV` in the `dotenv` config as done here?

https://github.com/gatsbyjs/gatsby/blob/cf3430462748cecdfb4b9f6b61285534fb4a2ea9/benchmarks/source-contentful/bin/site.js#L7

File: benchmarks/source-flotiq/scripts/updater.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
+const faker = require("faker")
```
Comment (duffn) [inline benchmarks/source-flotiq/scripts/updater.js:1]: Can you please run `yarn format` over the entire `source-flotiq` directory?

**Review by duffn — CHANGES_REQUESTED** [review]
I ran the 512 and 4096 benchmarks and they worked! 

I have a few small requests and then this should be good to merge.

**Review by duffn — APPROVED** [review]

**Comment by andrzejwp** [comment]
Thanks @duffn @moonmeister @pvdz for getting this into core. Looking forward to seeing our benchmarks on the site :-)

**Pull request merged**